tests/App/Entity/Gallery.php

```diff
@@ -15,10 +15,14 @@
 
 use Doctrine\ORM\Mapping as ORM;
 use Sonata\MediaBundle\Entity\BaseGallery;
+use Sonata\MediaBundle\Model\GalleryHasMediaInterface;
 
 /**
  * @ORM\Entity
  * @ORM\Table(name="media__gallery")
+ *
+ * @phpstan-template T of GalleryHasMediaInterface
+ * @phpstan-extends BaseGallery<T>
```

Comment:
Gallery related classes/interface are not generic unlike https://github.com/sonata-project/SonataMediaBundle/pull/2035/files#diff-f77d00a1d76f9856d3b4df2488bcd5de873c644cf5d7aa153ca791493e6d1674R23

Main reason: methods `setGalleryHasMedias` and similar doesn't exist in 4.x https://github.com/sonata-project/SonataMediaBundle/blob/4.x/src/Model/GalleryInterface.php